README.md

```diff
@@ -93,6 +93,23 @@ We use Code generation to generate our types for our GraphQL queries and mutatio
 - Each query and mutation should have a unique name.
 - Since query analysis for type generation occurs statically we cant place dynamic variables with in query strings we instead have to hard code the variable in the query or pass it in as query variable.
 
+## Testing
+Spruce has a combination of unit tests using Jest, and integration tests using Cypress.
+
+### Unit tests
+TODO: write more here. The Jest tests can be run by typing `yarn test`
+
+### Integration tests
+At a high level, we use Cypress to start a virtual browser that is running Spruce. Cypress then is able to run our test specs, which tell it to interact with the browser in certain ways and makes assertions about what happens in the UI. Note that you must be running the Evergreen server on localhost:9090 for the front-end to work.
+
+In order to run the Cypress tests, do the following, assuming you have this repo checked out and all the dependencies installed by yarn:
+1. Start the evergreen back-end with the sample local test data. You can do this by typing `make local-evergreen` in your evergreen folder.
+2. Start the Spruce dev server by typing `yarn dev` in this repo.
+3. Run Cypress by typing one of the following:
+    - `yarn cy:open` - opens the Cypress app in interactive mode. You can select tests to run from here in the Cypress browser.
```

Comment:
it may be helpful to note that the first option opens run tests against a version of chrome while the second option runs tests in a headless browser

Reply:
I'd like to omit this detail since a new engineer doesn't necessarily need to know the difference to use this initially